dls-controls/malcolmjs
# Feature/new table #212

## Description

Replaced old table component with new one based on react-virtualized, for greatly improved performance, behaviour and general structure

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA:SEQ1/table

## Agile board tracking

connect to #212
